engines/order_management/app/services/order_management/order/stripe_sca_payment_authorize.rb

```diff
@@ -15,7 +15,9 @@ def call!(redirect_url = full_order_path(@order))
 
         @payment.authorize!(redirect_url)
 
-        @order.errors.add(:base, I18n.t('authorization_failure')) unless @payment.pending?
+        unless @payment.pending? || @payment.requires_authorization?
```

Comment:
now that I saw the data migration, are we eventually removing the `@payment.pending?` clause?